schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/ConfluentSchemaRegistryCompatibleResource.java

```diff
@@ -479,25 +427,25 @@ public String toString() {
         }
     }
 
-    public static class Id {
-        private long id;
+    public static class VersionId {
```

Comment:
For register please leave id, as it isn't the version id, but actually the global unique id.

Reply:
Also if VersionId class is needed, can we rather call it just Version so it matches more the name used in confleunt's Schema object.

Reply:
Makes total sense.